src/java.base/share/native/libjimage/imageFile.cpp

```diff
@@ -212,6 +212,11 @@ ImageFileReaderTable::ImageFileReaderTable() : _count(0), _max(_growth) {
 }
 
 ImageFileReaderTable::~ImageFileReaderTable() {
+// Bug 8166727
+//
+// WARNING: Should never close the jimage file.
+//          Threads may still be running at shutdown.
+#if 0
```

Comment:
Are you keeping the code in order to re-visit it again? Just wondering why it's not deleted.

Reply:
Leaving the code as an example of what is required to close in case the topic gets revisited in the future and I get hit by a bus or dementia.

Reply:
Okay although I assume someone will spot this and be tempted to remove it.